src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -98,26 +106,38 @@ function makeColumnData(title: string, dataIndex: string, type: ColumnType, rend
 
 const AGG_COLUMN_TYPE = 'real';
 
-const mapColumn =
-  (mode: DisplayMode, bucketColumns: BucketColumn[]) => (column: AnonymizedResultColumn, columnIdx: number) => {
-    if (column.type === 'aggregate') {
-      switch (mode) {
-        case 'anonymized':
-          return [makeColumnData(column.name, columnIdx + '_anon', AGG_COLUMN_TYPE, renderValue)];
-        case 'combined':
-          return [
-            makeColumnData(column.name + ' (anonymized)', columnIdx + '_anon', AGG_COLUMN_TYPE, renderLowCountValue),
-            makeColumnData(column.name + ' (original)', columnIdx + '_real', AGG_COLUMN_TYPE, renderValue),
-            makeColumnData('Distortion', columnIdx + '_diff', 'real', renderRelativeNoiseValue),
-          ];
-      }
+const mapColumn = (mode: DisplayMode, bucketColumns: BucketColumn[]) => (column: AnonymizedResultColumn) => {
+  if (column.type === 'aggregate') {
+    switch (mode) {
+      case 'anonymized':
+        return [makeColumnData(column.name, 'anon', AGG_COLUMN_TYPE, renderValue)];
+      case 'combined':
+        return [
+          makeColumnData(column.name + ' (anonymized)', 'anon', AGG_COLUMN_TYPE, renderLowCountValue),
+          makeColumnData(column.name + ' (original)', 'real', AGG_COLUMN_TYPE, renderValue),
+          makeColumnData('Distortion', 'diff', 'real', renderRelativeNoiseValue),
+        ];
     }
+  }
 
-    return [makeColumnData(column.name, columnIdx.toString(), column.type, buildCellRenderer(column, bucketColumns))];
-  };
+  return [makeColumnData(column.name, column.name, column.type, buildCellRenderer(column, bucketColumns))];
+};
 
 // Rows
 
+// Represents data shown in the table; contains values which are displayed along
+// with some metadata used to style and render the row.
+type TableRowData = RowData & {
+  // RowData contains fields derived from bucket columns,
+  // which are shown in the table along with:
+  anon: number | null;
+  real: number;
+  diff: number | null;
```

Comment:
This is a step back, because with this we now only support one single aggregator per output row. The moment we add a different aggregator this will become inconsistent.